UnrealEngine/Plugins/JSBSimFlightDynamicsModel/Source/JSBSimFlightDynamicsModel/Public/JSBSimMovementComponent.h

```diff
@@ -197,6 +197,11 @@ class JSBSIMFLIGHTDYNAMICSMODEL_API UJSBSimMovementComponent : public UActorComp
 	UPROPERTY(Transient, BlueprintReadOnly, VisibleAnywhere, Category = "State")
 	FAircraftState AircraftState;
 
+	UPROPERTY(Transient, BlueprintReadOnly, VisibleAnywhere, Category = "Settings")
+	bool bUseExternalAircraftPath;
```

Comment:
```suggestion
	bool UseExternalAircraftPath;
```
JSBSim does not use [Hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation) nor does the Unreal plugin AFAICT.
Could you please stick to the naming convention of the plugin ?

Reply:
This is the Unreal Engine naming convention which is used in all the plugin. Everything is pascal case and boolean starts with b.

Reply:
Could not find any `bool` in the plugin nor any occurrence of this notation. Could you please point out one example to support your claim ?

Reply:
The plugin uses the Unreal naming convention, but I can change this variable name if it's not correct for the repository.

Reply:
> This is the Unreal Engine naming convention which is used in all the plugin.

> The plugin uses the Unreal naming convention

Hmm, I think I got the idea the first time you wrote it.

Could you please provide an example in the plugin ? Just one line in one file of this plugin and I set this conservation as resolved and it saves you the trouble of changing the variable name.

Reply:
Found it for you
https://github.com/JSBSim-Team/jsbsim/blob/92fa1eaa29322dc3f6ab55587161e55a6b24d36d/UnrealEngine/Plugins/JSBSimFlightDynamicsModel/Source/JSBSimFlightDynamicsModel/Private/JSBSimMovementComponent.cpp#L206

Reply:
The topic is resolved.